service/internal/auth/authn.go

```diff
@@ -145,6 +154,9 @@ func NewAuthenticator(ctx context.Context, cfg Config, logger *logger.Logger, we
 	a.publicRoutes = append(a.publicRoutes, cfg.PublicRoutes...)
 	a.publicRoutes = append(a.publicRoutes, allowedPublicEndpoints[:]...)
 
+	// Combine IPC reauthorization routes
+	a.ipcReauthRoutes = append(ipcReauthRoutes[:], cfg.IPCReauthRoutes...)
```

Comment:
intention is to append, not overwrite?

Reply:
Correct. This is to support extending the platform.